templates/docs/examples/patterns/x-small-capitalized

```diff
@@ -0,0 +1,6 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Align by baseline{% endblock %}
+
+{% block content %}
+<p class="p-text--x-small-capitalized">extra small capitalised text</p>
```

Comment:
These examples are not loading in the demo?

Reply:
fixed